### PR TITLE
Build a client without making a network call to Github

### PR DESCRIPTION
**Pull request by jarry-xiao**

### CODE REVIEW

File: typescript/phoenix-sdk/src/client.ts [hunk 1]
```diff
@@ -167,6 +167,45 @@ export class Client {
...
+      throw new Error("Unable to get clock");
+    }
+    const clock = deserializeClockData(clockBuffer);
+
+    const marketKeysToData: Array<[PublicKey, AccountInfo<Buffer>]> =
+      marketAddresses.map((marketAddress, index) => {
+        return [marketAddress, accounts[index] as AccountInfo<Buffer>];
+      });
+
+    const [markets, tokens] = Client.loadMarketsAndTokens(marketKeysToData, []);
```
Comment (throwbackjams) [inline typescript/phoenix-sdk/src/client.ts:191]: Should we add a comment here that notes the returned tokens will be empty and to use one of the other methods if that info is needed?

**Review by throwbackjams — APPROVED** [review]

File: typescript/phoenix-sdk/src/client.ts (hunk 1 above)
Comment (jarry-xiao) [reply typescript/phoenix-sdk/src/client.ts:191]: I don't think the contents will be empty, but the metadata should be. There are a lot of really wacky abstraction violations throughout this class IMO :(

Comment (throwbackjams) [reply typescript/phoenix-sdk/src/client.ts:191]: Ah I read that wrong. Yeah looks like the metadata fields will be just be a slice of the mint key for name and the full mint key for symbol

File: typescript/phoenix-sdk/src/client.ts [hunk 2]
```diff
@@ -167,6 +167,45 @@ export class Client {
...
+      "confirmed"
+    );
+
+    const clockBuffer = accounts.pop()?.data;
+    if (clockBuffer === undefined) {
+      throw new Error("Unable to get clock");
+    }
+    const clock = deserializeClockData(clockBuffer);
+
+    const marketKeysToData: Array<[PublicKey, AccountInfo<Buffer>]> =
```
Comment (eugene-chen) [inline typescript/phoenix-sdk/src/client.ts:186]: this field is misleadingly named; can we give it a better name, change it here, and also change it in `loadMarketsAndTokens`? name implies it's a map

File: typescript/phoenix-sdk/src/client.ts [hunk 3]
```diff
@@ -167,6 +167,45 @@ export class Client {
...
+  static async createWithoutConfig(
+    connection: Connection,
+    marketAddresses: PublicKey[],
+    trader?: PublicKey
+  ): Promise<Client> {
+    const accounts = await connection.getMultipleAccountsInfo(
+      [...marketAddresses, SYSVAR_CLOCK_PUBKEY],
+      "confirmed"
+    );
+
```
Comment (eugene-chen) [inline typescript/phoenix-sdk/src/client.ts:179]: toss in an assertion that `accounts.length === marketAddresses.length + 1`?

**Review by eugene-chen — APPROVED** [review]

**Pull request merged**